### ISSUE TITLE
Adding a way of matching the event with a moment in time within the tracking data

### ISSUE THREAD

**Issue by FCrSTATS** [PySport/kloppy#61]
There is a need to match the time of an event with the time within the tracking data, I am trying to implement a way this can be done accurately to remove the error syncing the datasets (opta,tracab). The first step to this is to add a very clumsy sync via the use of timestamps before things are refined. 

I would propose two attributes are added to the event object structure, clumsy_sync and precise_sync which are empty and only filled if the user wishes (as it relies on event data and tracking data to be present. 

filled by the following function.

Interested to hear thoughts 

<img width="913" alt="image" src="https://user-images.githubusercontent.com/35891964/94101815-f9bcb280-fe30-11ea-9598-ca731a54eb7d.png">

**Comment by koenvo** [comment]
This is a great use case of https://github.com/PySport/kloppy/issues/53 . This way you would be able to specify your own event class which has those fields

**Comment by koenvo** [comment]
I would consider adding this to the core of kloppy. The sync between an event and a frame where it's happening is very useful. To make this happen a "reference_frame" (type: Frame) attribute should be added to an Event

What do you think @bdagnino ?

**Comment by bdagnino** [comment]
Yes, sounds great and good approach. Would kloppy also provide methods better synchronize the data like proposed [here](https://www.kwiatkowski.io/sync.soccer) or at this stage it would be only to be able to reference between tracking and event data whatever timestamps are already in the data?

**Comment by koenvo** [comment]
Great! The idea is to implement something like @huffyhenry did. But I'm not totally sure what approach @FCrSTATS had in mind right now. Maybe you can elaborate a bit, @FCrSTATS ?
